Rakefile

```diff
@@ -18,6 +18,51 @@ task :check_test_env do
   raise 'RAILS_ENV not set to test' unless Rails.env.test?
 end
 
+task :build_indexes do
+  require 'scalarm/service_core/initializers/mongo_active_record_initializer'
+
+  MongoActiveRecordInitializer.start(Rails.application.secrets.database) unless Rails.env.test?
```

Comment:
@dkrol - być może jest to nadmiarowe, bo rake nie używamy w trybie test (w testach uruchamiane są tylko initializery)